docs/getting-started/provider-aws.md

```diff
@@ -0,0 +1,226 @@
+---
+title: AWS Quickstart
+weight: 2
+---
+
+Connect Crossplane to AWS to create and manage cloud resources from Kubernetes with the [AWS Official Provider](https://marketplace.upbound.io/providers/upbound/provider-aws).
```

Comment:
Providers are outside of the scope of the charter but they are inextricably linked to the usage of Crossplane. As a result discussion of using providers becomes a requirement for user documentation. I would encourage anyone in the community or any provider author to build and contribute a quickstart for their provider. If/when we need guidelines around the docs contributor's guide would be a great home for it.

Regarding the maintenance burden, these guides should be version independent (or pin to a minimum version). The idea isn't to change a quickstart with every release. The goal is to show basic functionality. If the number of contributed provider quick starts grows too large, that's a good problem to have and we can solve it then.  For now, I believe it is in the best interest of the community to provide the quickstart guides directly in the crossplane docs. If a doc author wishes to link to marketplace.upbound.io or any other external resource, that is fine and up to the author.

Reply:
Dan, do you feel our offline discussion resolved this?

Reply:
If it feels useful, you could de-emphasize provider docs in the `sitemap.xml` whilst still allowing them to be crawled and indexed.

Reply:
@plumbis I believe we already chatted about this offline, but if there was a _theoretical_ cloud provider that didn't require providing credentials (or distributed temporary creds for demoing), but gave you a unique URL where you could go see resources provisioned by Crossplane, would using it as the only provider in a quickstart be tenable in your mind? It would:
- Make the quickstart seamless and assuage any concerns about putting creds in a cluster
- Avoid needing to pick providers to use in the quickstart
- Avoid the maintenance burden of using multiple providers in the quickstart

Reply:
Possibly. A major value of having a quickstart for a specific provider instead of something abstract is it connects something the user knows (their cloud provider) to something new (Crossplane). It shows how the the new thing connects to and modifies the existing thing.  

This is why I don't think something like `provider-noop` would be effective. 

I would be 100% open to considering the theoretical provider you're describing but it needs to make a clear "magic moment" without requiring the user to learn another new technology.